tarodb/assets_store.go

```diff
@@ -762,6 +788,61 @@ func (a *AssetStore) FetchAllAssets(ctx context.Context,
 	return dbAssetsToChainAssets(dbAssets, assetWitnesses)
 }
 
+// FetchManagedUTXOs fetches all UTXOs we manage.
+func (a *AssetStore) FetchManagedUTXOs(ctx context.Context) (
+	[]*ManagedUTXO, error) {
+
+	var (
+		utxos []ManagedUTXORow
+		err   error
+	)
+
+	readOpts := NewAssetStoreReadTx()
+	dbErr := a.db.ExecTx(ctx, &readOpts, func(q ActiveAssetsStore) error {
+		utxos, err = q.FetchManagedUTXOs(ctx)
+		return err
+	})
+	if dbErr != nil {
+		return nil, dbErr
+	}
+
+	managedUtxos := make([]*ManagedUTXO, len(utxos))
+	for i, u := range utxos {
+		u := u
```

Comment:
I think this line can be removed.

Reply:
Yes! Fixed: https://github.com/lightninglabs/taro/pull/185/commits/101081501b7ea251efe84f9abe90388e683964fd